tests/test_console.py

```diff
@@ -39,3 +43,49 @@ def test_HBNBCommand_class_docstring(self):
                          "HBNBCommand class needs a docstring")
         self.assertTrue(len(HBNBCommand.__doc__) >= 1,
                         "HBNBCommand class needs a docstring")
+
+
+class TestHBNBCommandClass(TestCase):
+    """ Tests HBNBCommand class """
+
+    def test_do_create(self):
```

Comment:
This whole method is densely packed and hard to read. Maybe add some blank lines between the different chunks to help separate things.

Reply:
Can do! I separated by the comment blocks to make this more readable.